### ISSUE TITLE
Possible error in getGeoRectangle() method of MapUtils?

### ISSUE THREAD

**Issue by olneyMatt** [amcharts/amcharts5#141]
Hi,

Trying to use the getGeoRectangle() method from MapUtils and I can't get it draw properly.

Any attempt to get the rectangle returns a rectangle that is correct in longitude but extends from 89.999 to -89.999 in latitude.

![Screenshot 2021-12-13 at 13 35 44](https://user-images.githubusercontent.com/94979269/145822074-ee24ce37-f53b-4cd3-a31f-c75cd02224e3.png)

Checking the source code (source/.internal/charts/map/MapUtils) I think the error is from line 56:

```
if (west <= -180) {
   west = -179.9999;
}
if (south >= -90) {
   south = -89.9999;
}
if (north <= 90) {
   north = 89.9999;
}
if (east >= 180) {
   east = 179.9999;
}
```

I think the centre two `if` statements should be

```
if (south <= -90) {
   south = -89.9999;
}
if (north >= 90) {
   north = 89.9999;
}
```

Sorry - realise there's probably an easier way to tell you this but I'm new to git!

Cheers
Matt

**Comment by zeroin** [comment]
You are right, we will fix this issue in next release. Thanks!

**Comment by martynasma** [comment]
Heads up, this has been fixed in 5.0.18 just released.
